.github/workflows/publish-package.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     tags:
       - v*
+  schedule:
+    - cron: '30 8 * * 1'  # early morning (08:30 UTC) every Monday
```

Comment:
Just wonder if Sunday is better so that engineers in China time zone could use the new release on Monday.

Reply:
Good point. Will change to Sunday